### PR TITLE
GIDK-1115 dev | Rohit | Descriptions is added and Audit log issue resolved

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/audit-logs/audit-logs.component.ts [hunk 1]
```diff
@@ -143,4 +145,17 @@ export class AuditLogsComponent implements OnInit, OnDestroy {
...
+
+     /**
+     * This function will set the page heading
+     *
+     * @param {string} title
+     * @memberof AuditLogsFormComponent
+     */
+    public setCurrentPageTitle(title: string) : void {
+        let currentPageObj = new CurrentPage();
+        currentPageObj.name = title
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/audit-logs/audit-logs.component.ts:157]: add semicolon

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html [hunk 2]
```diff
@@ -281,12 +345,20 @@ <h1>{{log.operationType}}
                         </label>
                             <p class="ml-05"> {{log?.details?.total |giddhCurrency }}</p>
                         </div>
+                        <div class="mr-05" *ngIf="log?.details?.description">
+                            <label>
+                           description:
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html:350]: Replace small d with Capital D

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html [hunk 3]
```diff
@@ -160,6 +161,69 @@ <h1>{{log.operationType}}
 
         <div class="address-row-wrapper d-flex">
             <div class="address-row mt-05">
+                <!-- account address block start -->
+                <ng-container *ngIf="log && log?.details && log?.details?.addresses && log?.details?.addresses.length">
+                    <ng-container *ngFor="let item of log?.details?.addresses; let index = index">
+                        <div class="single-address-row" *ngIf="index===0">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html:167]: HTML repetition can be avoided with 
`<div class="single-address-row" *ngIf="index===0 || index>0 && showSingleAddress && isShowMultipleDataIndex===parentIndex">`

File: apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html [hunk 4]
```diff
@@ -501,6 +578,69 @@ <h1>{{log.operationType}}
 
             <div class="address-row-wrapper d-flex">
                 <div class="address-row mt-05">
+                    <!-- old account address start -->
+                    <ng-container *ngIf="log?.old && log?.old?.details && log?.old?.details?.addresses && log?.old?.details?.addresses.length">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/audit-logs/components/audit-logs-table/audit-logs-table.component.html:582]: HTML repetition can be avoided with:
`<div class="single-address-row" *ngIf="index===0 || index>0 && showSingleAddress && isShowMultipleDataIndex===parentIndex">`

**Pull request merged**